react/components/Combobox.tsx

```diff
@@ -55,7 +55,7 @@ export const ComboboxOption: React.FC<ReachComboboxOptionProps & {
     className={classNames(
       className,
       styles.option,
-      'flex items-center pointer bg-action-secondary pv3 ph5 hover-bg-action-secondary'
+      'flex items-center pv3 ph5 pointer bg-action-secondary hover-bg-action-secondary active-bg-action-secondary'
```

Comment:
Now this is "incorrect", but as soon as Styleguide bumps the Tachyons version, I'll open a PR bumping the Styleguide version here and it will have the expected color.

Reply:
you don't actually need to bump styleguide here after deploying the new version with `vtex-tachyons` updated, because the IO infra will automatically bump the styleguide version on the account itself when it is published.